tests/kuttl/test-suites/default/watcher-decision-engine/03-deploy-decision-engine.yaml

```diff
@@ -1,9 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: metric-storage-prometheus-config
+  namespace: watcher-kuttl-default
+stringData:
+  host: metric-storage-prometheus.watcher-kuttl-default.svc
+  port: "9090"
+  ca_secret: "combined-ca-bundle"
+  ca_key: "internal-ca-bundle.pem"
+---
 apiVersion: watcher.openstack.org/v1beta1
 kind: WatcherDecisionEngine
 metadata:
-  name: watcherdecisionengine-kuttl
+  name: watcher-decision-engine-kuttl
   namespace: watcher-kuttl-default
 spec:
   secret: osp-secret
```

Comment:
This shold be watcher-kuttl which is the secret created by the Watcher instance created in step 02.

Reply:
thank you